test/ci.sh

```diff
@@ -4,7 +4,7 @@ set -eo pipefail
 
 root_dir=$(pwd)
 
-flags="${@}"
+flags="use-werror=true ${@}"
```

Comment:
Yes, and I realized that right after I opened the PR :)  Please see the the
new commit.

On Mon, Feb 15, 2016 at 3:52 PM, Joshua Warner <notifications@github.com>
wrote:

> In test/ci.sh
> <https://github.com/ReadyTalk/avian/pull/476#discussion_r52951112>:
>
> > @@ -4,7 +4,7 @@ set -eo pipefail
> >
> >  root_dir=$(pwd)
> >
> > -flags="${@}"
> > +flags="use-werror=true ${@}"
>
> Did you mean use-werror=true instead?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ReadyTalk/avian/pull/476/files#r52951112>.
>